From a19f1d9f2d8a9070cd9ea3ed29c4f4fdf50309af Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 11 Aug 2026 09:50:32 +0000
Subject: [PATCH] SystemUI: Fix QS header clock color

[Port to A16] onColorsChanged(boolean) still present but uncalled in A16 (header uses setTextAppearance); removed method + now-unused imports.

Change-Id: Ie4d3a5597fed55e0d35a40b0282812c089a60872
---
 .../src/com/android/systemui/statusbar/policy/Clock.java | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java b/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java
index 97c71b2c319b..8cabd9b17376 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java
@@ -41,12 +41,10 @@ import android.text.format.DateFormat;
 import android.text.style.CharacterStyle;
 import android.text.style.RelativeSizeSpan;
 import android.util.AttributeSet;
-import android.view.ContextThemeWrapper;
 import android.view.Display;
 import android.view.View;
 import android.widget.TextView;
 
-import com.android.settingslib.Utils;
 import com.android.settingslib.applications.InterestingConfigChanges;
 import com.android.systemui.Dependency;
 import com.android.systemui.FontSizeUtils;
@@ -395,13 +393,6 @@ public class Clock extends TextView implements
         setTextColor(mNonAdaptedColor);
     }
 
-    // Update text color based when shade scrim changes color.
-    public void onColorsChanged(boolean lightTheme) {
-        final Context context = new ContextThemeWrapper(mContext,
-                lightTheme ? R.style.Theme_SystemUI_LightWallpaper : R.style.Theme_SystemUI);
-        setTextColor(Utils.getColorAttrDefaultColor(context, R.attr.wallpaperTextColor));
-    }
-
     public void onDensityOrFontScaleChanged() {
         ShadeWindowGoesAround.assertInLegacyMode();
         // Note that this class is not being registered as configuration listener when used
-- 
2.43.0

